public/index.html

```diff
@@ -0,0 +1,36 @@
+<!--
```

Comment:
Can't we re-use the existing Jinja2 template by having the Python app bootstrapping process just render the template as this file?